src/analysis/normed/field/basic.lean

```diff
@@ -414,6 +414,12 @@ instance normed_division_ring.to_norm_one_class : norm_one_class α :=
 ⟨mul_left_cancel₀ (mt norm_eq_zero.1 (one_ne_zero' α)) $
   by rw [← norm_mul, mul_one, mul_one]⟩
 
+instance normed_division_ring.is_absolute_value : is_absolute_value (norm : α  → ℝ) :=
```

Comment:
Is this true in a normed ring?

Reply:
No, `abv_mul` fails because normed ring only satisfy ‖x y‖ ≤ ‖x‖ ‖y‖.

Reply:
Either way,
```suggestion
instance is_absolute_value norm : is_absolute_value (norm : α  → ℝ) :=
```